terraform/modules/rds_migration/security_groups.tf

```diff
@@ -0,0 +1,27 @@
+resource "aws_security_group" "ecs" {
+  name        = "${var.prefix}-db-migration-ecs"
+  description = "DB migration ECS security group"
+  vpc_id      = var.vpc_id
+
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+resource "aws_vpc_security_group_ingress_rule" "db_migration_ecs_ingress" {
+  description                  = "Allow ingress on port ${var.database_port} of the db from the db migration ecs"
+  ip_protocol                  = "tcp"
+  from_port                    = var.database_port
+  to_port                      = var.database_port
+  referenced_security_group_id = aws_security_group.ecs.id
+  security_group_id            = var.db_security_group_id
+}
+
+resource "aws_vpc_security_group_egress_rule" "general_egress" {

```

Comment:
The `cf conduit` plugin seems to consistently use port 7080 on my machine in order to connect with the DB service when going to make a `pg dump` file. However on ECS it seems to be using a different port and I haven't been able to work out which and if it consistently uses that. I've tried a rule with all ports 7080 and above based on the conduit logic [here](https://github.com/alphagov/paas-cf-conduit/blob/b16106aa5089599280711403773f130b9493aeb0/main.go#L82), but that didn't work either! It didn't seem worth the time to try and pin point.

As a result, I've had to allow egress to all ports in order to make conduit work 😥 (this also allows ECS to connect to the DB and get secrets, otherwise there would be separate rules for these). I think this is ok given we'll run this on an ad-hoc basis a handful of times, and destroy the infra when we're not migrating from PaaS